### ISSUE TITLE
Dark-mode accessibility contrast

### ISSUE THREAD

**Issue by kjrsten** [BanchanArt/banchan#611]
Color combination for disabled button is not accessible (see screenshot)
<img width="491" alt="Screen Shot 2023-08-13 at 2 09 28 PM" src="https://github.com/BanchanArt/banchan/assets/5471856/31425f68-dbe3-4b82-a4a0-f87897f82e77">

**Comment by zkat** [comment]
Closing as duplicate of https://github.com/BanchanArt/banchan/issues/430
